liquibase-core/src/main/java/liquibase/sqlgenerator/core/CreateProcedureGenerator.java

```diff
@@ -19,9 +19,13 @@
 import liquibase.util.StringUtils;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 public class CreateProcedureGenerator extends AbstractSqlGenerator<CreateProcedureStatement> {
+
+    private static final List<String> BEFORE_BODY_START_STATEMENTS = Arrays.asList("CREATE", "ALTER");
```

Comment:
We use the constant only in one place, what about removing it?

Reply:
Done